CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the docker cookbook.
 
+## 4.6.6 (unreleased)
```

Comment:
In the future we have tooling for the changelog and the metadata version bump so this isn't necessary.

Reply:
I was not sure about that but will keep this in mind for future PRs ;)